src/api/telemetry/WebSocketWorker.js

```diff
@@ -335,14 +349,16 @@ export default function installWorker() {
      */
     readyForNextBatch() {
       if (this.#hasBatch) {
-        this.#sendNextBatch();
+        this.#throttledSendNextBatch();
       } else {
         this.#readyForNextBatch = true;
       }
     }
     #sendNextBatch() {
       const batch = this.#batch;
+      batch.number = Math.round(Math.random() * 100000);
       this.#resetBatch();
+      this.sent = Date.now();
```

Comment:
Doesn't look like `this.sent` is being used. Please remove it.

Reply:
Will do, thanks